specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json

```diff
@@ -121,7 +124,10 @@
             "in": "path",
             "description": "The requested administrator name.",
             "required": true,
-            "type": "string"
+            "type": "string",
+			"enum": [
+				"ActiveDirectory"
+			]
```

Comment:
Enums should specify the `x-ms-enum` extension documented at [https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum](https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum).

Reply:
This applies to all enums that were added.

---
In reply to: [342237499](https://github.com/Azure/azure-rest-api-specs/pull/7377#discussion_r342237499) [](ancestors = 342237499)